pom.xml

```diff
@@ -138,13 +138,10 @@
   </distributionManagement>
 
   <repositories>
-    <repository>
-      <id>terracotta-snapshots</id>
-      <url>http://www.terracotta.org/download/reflector/snapshots</url>
-    </repository>
     <repository>
       <id>terracotta-releases</id>
-      <url>http://www.terracotta.org/download/reflector/releases</url>
+      <url>https://repo.terracotta.org/maven2</url>
+      <snapshots><enabled>false</enabled></snapshots>
```

Comment:
Should we make this formatting consistent across projects?

Reply:
If it matters to you, I can do that -- my preference is for the more compact form ... it's not like you can put anything else in the `<snapshots>` or `<enabled>` elements.

Reply:
meh